tests/Api/ApiAdressesTest.php

```diff
@@ -168,14 +167,16 @@ public function test_address_post()
             'street' => 'street',
             'postal_code' => '12345',
         ]);
+
+        $address_id = $response->json('data.id');
+        $this->assertGreaterThan(0, $address_id);
```

Comment:
If we want to be picky, it should be `addressId`, not `address_id` as it's not [PSR-2 compliant](https://www.php-fig.org/psr/psr-2/).

Reply:
I agree with you: you're too picky ;)